appserver/core/api-exporter-fragment/pom.xml

```diff
@@ -149,11 +149,11 @@
                                     javax.lang.model.element; \
                                     javax.lang.model.type; \
                                     javax.lang.model.util; \
-                                    javax.mail; \
-                                    javax.mail.event; \
-                                    javax.mail.internet; \
-                                    javax.mail.search; \
-                                    javax.mail.util; \
+                                    jakarta.mail; \
+                                    jakarta.mail.event; \
+                                    jakarta.mail.internet; \
+                                    jakarta.mail.search; \
+                                    jakarta.mail.util; \
```

Comment:
We're not that far, are we? The packages are still the same.

Reply:
No, this is the result of me doing a sed and not thinking